modules/migrations/gogs.go

```diff
@@ -220,12 +229,45 @@ func (g *GogsDownloader) getIssues(page int, state string) ([]*base.Issue, bool,
 		if issue.PullRequest != nil {
 			continue
 		}
+		g.issueIndexes[issue.Index] = true
 		allIssues = append(allIssues, convertGogsIssue(issue))
 	}
 
 	return allIssues, len(issues) == 0, nil
 }
 
+// getPullRequests returns pull requests as issue
+func (g *GogsDownloader) getPullRequests(page int) ([]*base.Issue, bool, error) {
+	index := int64(page)
+	if g.issueIndexes[index] {
+		// next index
+		return nil, false, nil
+	}
+
+	gogsIssue, err := g.client.GetIssue(g.repoOwner, g.repoName, index)
+	if err != nil {
+		if err.Error() == "404 Not Found" {
+			return nil, true, nil
+		}
+		return nil, false, err
+	}
+
+	issue := convertGogsIssue(gogsIssue)
+
+	issue.Title = "[Pull] " + issue.Title
```

Comment:
I think we can actually just create PRs with missing heads or even just make fake ones ourselves

Reply:
we have no base and no head

Reply:
:point_up: with this all you get is a broken pull

Reply:
@Cat7373 you as one who potentially use this migrator ... what are your thoughts about it?

Reply:
@zeripath tested to create pull with only issue information: get `failed: Unsupported scheme: `

Reply:
yeah I forgot we don't have a base. We have a head it's simply refs/pulls/xxx/head - the base is the issue.